test/integrations/validation/Validator.spec.ts

```diff
@@ -226,6 +226,62 @@ describe('Validator', () =>
         });
     });
 
+    describe('Url values', () =>
+    {
+        it('should reject an invalid url with protocol', () =>
+        {
+            const input = { url: 'http://this is an invalid url' };
+            const test = () => validator.validate(input, SCHEMAS.URL_HTTP_HTTPS);
+
+            expect(test).toThrowError(ERRORS.URL);
+        });
+
+        it('should reject a url without protocol', () =>
+        {
+            const input = { url: 'masking.tech/images/peter.jpg' };
```

Comment:
```suggestion
            const input = { url: 'example.com/somefolder/somefile.ext' };
```

I think we should avoid using our own web address and start using fake addresses instead.
Can you do this for all occurrences below?